flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -257,6 +257,18 @@
 					text("\"ip\" - uses host's ip address as binding address"))
 				.build());
 
+	/**
+	 * The config options defined the prefix of the TaskManager's ResourceID. If configured, the ResourceID will be spliced
+	 * with the prefix and an uuid. Otherwise, the ResourceID would be a random hex string. Only valid in standalone mode.
+	 */
+	@Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER)
+	public static final ConfigOption<String> TASK_MANAGER_RESOURCE_ID_PREFIX =
+		key("taskmanager.resource-id.prefix")
+			.stringType()
+			.noDefaultValue()
+			.withDescription("The config options defined the prefix of the TaskManager's ResourceID. If configured, the ResourceID "
```

Comment:
"The prefix of the TaskManager's ResourceID. If configured, the ResourceID will be generated with the prefix and a 6-character random string. Otherwise, a 32-character random string will be used. Only valid in standalone mode."